### PR TITLE
feat: `avm/res/net-app/net-app-account`

### PR DESCRIPTION
**Pull request by segraef**
[avm.res.net-app.net-app-account](https://github.com/segraef/bicep-registry-modules/actions/workflows/avm.res.net-app.net-app-account.yml)

### CODE REVIEW

**Review by AlexanderSehr — CHANGES_REQUESTED** [review]
Requires uncommenting in the `/.github/CODEOWNERS` file

**Comment by segraef** [comment]
> Requires uncommenting in the `/.github/CODEOWNERS` file

done

File: avm/res/net-app/net-app-account/main.bicep [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+@description('Optional. Location for all resources.')
+param location string = resourceGroup().location
+
+@description('Optional. The lock settings of the service.')
+param lock lockType
+
+@description('Optional. Tags for all resources.')
+param tags object?
+
+@description('Optional. Enable telemetry via a Globally Unique Identifier (GUID).')
```
Comment (AlexanderSehr) [inline avm/res/net-app/net-app-account/main.bicep:45]:
```suggestion
@description('Optional. Enable/Disable usage telemetry for module.')
```

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -74,7 +74,7 @@
 /avm/res/managed-identity/user-assigned-identity/ @Azure/avm-res-managedidentity-userassignedidentity-module-owners-bicep @Azure/avm-core-team-technical-bicep
 #/avm/res/managed-services/registration-definition/ @Azure/avm-res-managedservices-registrationdefinition-module-owners-bicep @Azure/avm-core-team-technical-bicep
 #/avm/res/management/management-group/ @Azure/avm-res-management-managementgroup-module-owners-bicep @Azure/avm-core-team-technical-bicep
-#/avm/res/net-app/net-app-account/ @Azure/avm-res-netapp-netappaccount-module-owners-bicep @Azure/avm-core-team-technical-bicep
+/avm/res/net-app/net-app-account/ @Azure/avm-res-netapp-netappaccount-module-owners-bicep @Azure/avm-core-team-technical-bicep
```
Comment (eriqua) [inline .github/CODEOWNERS:77]: Please ref to https://azure.github.io/Azure-Verified-Modules/specs/shared/#team-memberships

File: .github/workflows/avm.res.net-app.net-app-account.yml [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+concurrency:
+  group: ${{ github.workflow }}
+
+jobs:
+  ###########################
+  #   Initialize pipeline   #
+  ###########################
+  job_initialize_pipeline:
+    runs-on: ubuntu-20.04
```
Comment (eriqua) [inline .github/workflows/avm.res.net-app.net-app-account.yml:47]:
```suggestion
    runs-on: ubuntu-latest
```

File: .github/CODEOWNERS (hunk 2 above)
Comment (segraef) [reply .github/CODEOWNERS:77]: @Azure/avm-res-netapp-netappaccount-module-owners-bicep
@Azure/avm-res-netapp-netappaccount-module-contributors-bicep

File: avm/res/net-app/net-app-account/capacity-pool/main.bicep [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+
+@description('The resource ID of the Capacity Pool.')
+output resourceId string = capacityPool.id
+
+@description('The name of the Resource Group the Capacity Pool was created in.')
+output resourceGroupName string = resourceGroup().name
+
+@description('The location the resource was deployed into.')
+output location string = capacityPool.location
+// =============== //
```
Comment (AlexanderSehr) [inline avm/res/net-app/net-app-account/capacity-pool/main.bicep:121]:
```suggestion

// =============== //
```

File: avm/res/net-app/net-app-account/capacity-pool/volume/main.bicep [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+
+@description('The Resource ID of the Volume.')
+output resourceId string = volume.id
+
+@description('The name of the Resource Group the Volume was created in.')
+output resourceGroupName string = resourceGroup().name
+
+@description('The location the resource was deployed into.')
+output location string = volume.location
+// =============== //
```
Comment (AlexanderSehr) [inline avm/res/net-app/net-app-account/capacity-pool/volume/main.bicep:101]:
```suggestion

// =============== //
```

File: avm/res/net-app/net-app-account/tests/e2e/waf-aligned/main.test.bicep [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+// Parameters //
+// ========== //
+
+@description('Optional. The name of the resource group to deploy for testing purposes.')
+@maxLength(90)
+// e.g., for a module 'network/private-endpoint' you could use 'dep-dev-network.privateendpoints-${serviceShort}-rg'
+param resourceGroupName string = 'dep-${namePrefix}-netapp.netappaccounts-${serviceShort}-rg'
+
+@description('Optional. The location to deploy resources to.')
+param location string = 'westus3'
```
Comment (AlexanderSehr) [inline avm/res/net-app/net-app-account/tests/e2e/waf-aligned/main.test.bicep:16]: Will be overwritten. Please refer to https://github.com/Azure/bicep-registry-modules/pull/792#discussion_r1455072261

**Review by AlexanderSehr — DISMISSED** [review]

File: avm/res/net-app/net-app-account/tests/e2e/waf-aligned/main.test.bicep (hunk 6 above)
Comment (segraef) [reply avm/res/net-app/net-app-account/tests/e2e/waf-aligned/main.test.bicep:16]: adjusted with tempLocation

**Review by AlexanderSehr — APPROVED** [review]

**Pull request merged**